ansible/idr-proxy.yml

```diff
@@ -49,14 +49,22 @@
         }}:8080
     when: "{{ groups[idr_environment | default('idr') + '-a-dockermanager-hosts'] is defined }}"
 
-  - name: Configure rsync port in selinux
+  - name: Install packages for selinux
     become: yes
-    seport:
-      ports: 873
-      proto: tcp
-      setype: http_port_t
+    yum:
+      name: "policycoreutils-python"
       state: present
 
+  - name: Configure rsync port in selinux
+    become: yes
+    command: semanage port -m -t http_port_t -p tcp 873
```

Comment:
Which version of Ansible? Once all the other changes are in if it still doesn't work can you add a trello card?